src/survey_metadata.js

```diff
@@ -43,26 +49,67 @@ var surveyJSON = {
      "visibleIf": "{confirm_state_ca} = 'n'",
      "title": "Tell us which state or territory you live in",
      "choices": [
-      {
-       "value": "WA",
-       "text": "Washington"
-      },
-      {
-       "value": "OR",
-       "text": "Oregon"
-      },
-      {
-       "value": "NV",
-       "text": "Nevada"
-      }
+      "Alabama (AL)",

```

Comment:
I think that these should also have "value".  E.g. instead of:

```
"Alabama (AL)",
```
it would be
```
{
  "value": "AL",
  "text": "Alabama (AL)"
},
```

The SurveyJS UI lets you set those elements- you don't have to do it in the code...